src/core/action-core.js

```diff
@@ -31,11 +32,19 @@ function createAction(action) {
 
         action.id = rows[0].id;
         if (action.type === 'IMAGE' || action.type === 'TEXT') {
-          return createFeedItem(action, trx);
+          createFeedItem(action, trx);
         }
 
         return Promise.resolve();
-      });
+    })
+  })
+  .then(() => undefined)
+  .catch(err => {
+    if (err.constraint === 'only_one_check_in_per_event') {
```

Comment:
this string COULD be defined somewhere as a constant, and you could use that definition in here & in the index creation as well... but not a blocker